doc/maintaining/data-viewer.rst

```diff
@@ -1,223 +1,332 @@
-===========
-Data Viewer
-===========
+==============================
+Data preview and visualization
+==============================
 
+.. versionchanged:: 2.3
 
-.. todo:: This whole page is outdated and needs to be updated for the new Resource Views
+    The whole way resource previews are handled was changed on CKAN 2.3.
+    Please refer to previous versions of the documentation if you are using
+    an older CKAN version.
 
+.. contents::
 
-The CKAN resource page can contain a preview of the resource's data.
-This works by either:
 
-1. Embedding the data into the page, either directly or by loading the data
-   in an iframe.
-2. Using a custom widget (such as `Recline <http://okfnlabs.org/recline>`_)
-   to view the data.
 
-Generally, the decision as to which action to take is determined by the type of
-resource being viewed.
-In general, images will be directly embedded, unstructured or plain text
-files will be loaded in an iframe, and more complex data types will need to
-use a custom widget.
+Overview
+--------
 
-The data preview functionality that is provided by CKAN is described in
-the following sections:
+The CKAN resource page can contain one or more views of the resource data
+or file contents. These are commonly referred to as *resource views*.
 
-* :ref:`image-preview`
-* :ref:`data-explorer`
-* :ref:`text-preview`
-* :ref:`pdf-preview`
-* :ref:`resource-proxy`
-* :ref:`embed-previews`
+.. image:: /images/views_overview.png
 
-These sections list the resource formats that each extension can preview and
-provide instructions for how to enable each extension.
-It is also possible for developers to create new extensions that can preview
-different types of resources.
-For more information on this topic see
-`Writing Extensions <extensions/index>`_.
+The main features of resource views are:
 
+* One resource can have multiple views of the same data (for example a grid
+  and some graphs for tabular data).
 
-.. _image-preview:
+* Dataset editors can choose which views to show, reorder them and configure
+  them individually.
 
-Viewing images and text files
------------------------------
+* Views configuration persits on the database, so it's not lost on each page
+  refresh.
```

Comment:
I might just delete this. Of course it persists, that's what people would assume. Of course, it didn't used to persist. But users don't need to know about that.

(The right place for pointing out "it didn't used to persist but now it does" would be in places like changelog, release notes or announcement, feature announcement blog post.)